legal-copy/privacy-notice.html

```diff
@@ -0,0 +1,84 @@
+
```

Comment:
This is what `gulp legal` generates, no? Should probably not be checked in, since it's a build artifact